util/src/main/DocTaglet.java

```diff
@@ -14,57 +14,46 @@
  * limitations under the License.
  */
 
-import com.sun.javadoc.Tag;
-import com.sun.tools.doclets.Taglet;
+import com.sun.source.doctree.DocTree;
+import com.sun.source.doctree.UnknownBlockTagTree;
+import jdk.javadoc.doclet.Taglet;
+import javax.lang.model.element.Element;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 
-public abstract class DocTaglet implements Taglet {
-
-    public boolean inConstructor() {
-        return true;
-    }
-
-    public boolean inField() {
-        return true;
-    }
-
-    public boolean inMethod() {
-        return true;
-    }
 
-    public boolean inOverview() {
-        return true;
-    }
-
-    public boolean inPackage() {
-        return true;
-    }
+public abstract class DocTaglet implements Taglet {
 
-    public boolean inType() {
-        return true;
+    @Override
+    public Set<Location> getAllowedLocations() {
+        Set<Location> allowedLocations = new HashSet<Location>();
+        allowedLocations.addAll(Arrays.asList(Location.CONSTRUCTOR, Location.METHOD, Location.FIELD, Location.OVERVIEW, Location.PACKAGE,
+                Location.TYPE));
+        return allowedLocations;
     }
 
+    @Override
     public boolean isInlineTag() {
         return false;
     }
 
-    public String toString(final Tag[] tags) {
-        if (tags.length == 0) {
+    @Override
+    public String toString(List<? extends DocTree> tags, Element element) {
+        if (tags.size() == 0) {
             return null;
         }
-
         StringBuilder buf = new StringBuilder(String.format("%n<dl><dt><span class=\"strong\">%s</span></dt>%n", getHeader()));
-        for (Tag t : tags) {
-            buf.append("   <dd>").append(genLink(t.text())).append("</dd>%n");
+        for (DocTree t : tags) {
+            String text = ((UnknownBlockTagTree) t).getContent().get(0).toString();
+            buf.append("   <dd>").append(genLink(text)).append("</dd><br/>");
```

Comment:
Is this what's creating the extra white below the MongoDB Documentation and Since Server Release paragraphs,  as seen in the first screen shot?  And #468 will fix it?

Reply:
Yes ! I suggest to merge #468 on master. I will rebase and fix conflict on this line

Reply:
Merged!